controllers/provisioners/eks/scaling/launchtemplate_test.go

```diff
@@ -0,0 +1,611 @@
+						Arn: aws.String("arn::aws:my-role"),
+					},
+					SecurityGroupIds: aws.StringSlice([]string{}),
+				},
+			},
+			input: &CreateConfigurationInput{
+				SecurityGroups:        []string{},
+				IamInstanceProfileArn: "arn::aws:other-role",
+			},
+			shouldDrift: true,
+		},
+		{
+			launchTemplate: &ec2.LaunchTemplate{
+				LaunchTemplateName: aws.String("my-launch-config"),
+			},
+			latestVersion: &ec2.LaunchTemplateVersion{
+				LaunchTemplateData: &ec2.ResponseLaunchTemplateData{
+					IamInstanceProfile: &ec2.LaunchTemplateIamInstanceProfileSpecification{
+						Arn: aws.String(""),
+					},
+					SecurityGroupIds: aws.StringSlice([]string{"sg-1"}),
+				},
+			},
+			input: &CreateConfigurationInput{
+				SecurityGroups: []string{"sg-2"},
+			},
+			shouldDrift: true,
+		},
+		{
+			launchTemplate: &ec2.LaunchTemplate{
+				LaunchTemplateName: aws.String("my-launch-config"),
+			},
+			latestVersion: &ec2.LaunchTemplateVersion{
+				LaunchTemplateData: &ec2.ResponseLaunchTemplateData{
+					IamInstanceProfile: &ec2.LaunchTemplateIamInstanceProfileSpecification{
+						Arn: aws.String(""),
+					},
+					SecurityGroupIds: aws.StringSlice([]string{}),
+					KeyName:          aws.String("other-key"),
+				},
+			},
+			input: &CreateConfigurationInput{
+				SecurityGroups: []string{},
+				KeyName:        "my-key",
+			},
+			shouldDrift: true,
+		},
+		{
+			launchTemplate: &ec2.LaunchTemplate{
+				LaunchTemplateName: aws.String("my-launch-config"),
+			},
+			latestVersion: &ec2.LaunchTemplateVersion{
+				LaunchTemplateData: &ec2.ResponseLaunchTemplateData{
+					IamInstanceProfile: &ec2.LaunchTemplateIamInstanceProfileSpecification{
+						Arn: aws.String(""),
+					},
+					SecurityGroupIds: aws.StringSlice([]string{}),
+					UserData:         aws.String("hello"),
+				},
+			},
+			input: &CreateConfigurationInput{
+				SecurityGroups: []string{},
+				UserData:       "test",
+			},
+			shouldDrift: true,
+		},
+		{
+			launchTemplate: &ec2.LaunchTemplate{
+				LaunchTemplateName: aws.String("my-launch-config"),
+			},
+			latestVersion: &ec2.LaunchTemplateVersion{
+				LaunchTemplateData: &ec2.ResponseLaunchTemplateData{
+					IamInstanceProfile: &ec2.LaunchTemplateIamInstanceProfileSpecification{
+						Arn: aws.String(""),
+					},
+					SecurityGroupIds: aws.StringSlice([]string{}),
+					BlockDeviceMappings: []*ec2.LaunchTemplateBlockDeviceMapping{
+						{
+							DeviceName: aws.String("/dev/xvda"),
+							Ebs: &ec2.LaunchTemplateEbsBlockDevice{
+								VolumeType: aws.String("gp2"),
+								VolumeSize: aws.Int64(30),
+							},
+						},
+					},
+				},
+			},
+			input: &CreateConfigurationInput{
+				SecurityGroups: []string{},
+				Volumes: []v1alpha1.NodeVolume{
+					{
+						Name: "/dev/xvda",
+						Type: "gp2",
+						Size: 100,
+					},
+				},
+			},
+			shouldDrift: true,
+		},
+	}
+
+	for i, tc := range tests {
+		t.Logf("Test #%v", i)
```

Comment:
Is there a reason you don't use `testing.Run()` ?

I was thinking maybe adding a `name: string` or `description: string` to the tests struct that can describe the test case.

then the output becomes:
```
TestLaunchTemplateDrifted/volume-size-should-drift    ok
```